.github/workflows/quality.yml

```diff
@@ -74,7 +118,6 @@ jobs:
         run: ./gradlew clean lint${{ matrix.target }}Release --stacktrace
       - name: Upload ${{ matrix.target }} linting report
         uses: actions/upload-artifact@v2
-        if: always()
```

Comment:
Just a last remark, will the lint report be uplodoad if the command fails (so if there are lint issue) if you remove this line?

Reply:
(related to your comment https://github.com/vector-im/element-android/pull/4864/files#r779175980)

Reply:
No, without ` if: always()` the artifact will not be uploaded. While if a step fails all the subsequent ones will not run.

Reply:
Edit:
Here is a different case than before because the way command `./gradlew clean lint` run needs to be succeed in order to generate the report! So there is no need to upload an artifact without the report. In comparison to `ktlint`
that the report is generated when the build fails.

Reply:
Yes, thanks!